### PR TITLE
improve aethestics/view of method configs listing

### PR DESCRIPTION
**Pull request by eddiebroad**
can someone review the code and the results of this code/commit?
I am wondering about people's thoughts/opinions, not only on the code
but also of its effects on the aesthetics/view and also about whether some columns should be dropped.

### CODE REVIEW

**Comment by MattPutnam** [comment]
I'm not going to worry about the number of columns for now.  I think the style of the table is going to change, and that will help the problem, and it should be easy enough to add or remove columns at any time.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 1]
```diff
@@ -13,33 +13,40 @@
...
-         :workspaceName {:namespace (str "*ws_ns_" (inc i))
-                        :name (str "*ws_n_" (inc i))}
-         :methodStoreMethod {:namespace (str "*ms_ns_" (inc i))
-                            :name (str "*ms_n_" (inc i))
-                            :version (str "*ms_v_" (inc i))}
-         :methodStoreConfig {:namespace (str "*msc_ns_" (inc i))
-                             :name (str "*msc_n_" (inc i))
-                             :version (str "*msc_v_" (inc i))
+         :workspaceName {:namespace (str "ws_ns_" (inc i))
+                        :name (str "ws_n_" (inc i))}
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: Nit: check alignment (lots of other places too)

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 2]
```diff
@@ -13,33 +13,40 @@
...
+         :methodStoreMethod {:methodNamespace (str "ms_ns_" (inc i))
+                            :methodName (str "ms_n_" (inc i))
+                            :methodVersion (str "ms_v_" (inc i))}
+         :methodStoreConfig {:namespace (str "msc_ns_" (inc i))
+                             :name (str "msc_n_" (inc i))
+                             :version (str "msc_v_" (inc i))
                              }
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: This shouldn't have its own line

Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: exactly what shouldn't have its own line?

Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: you mean the "}" on line 24?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 3]
```diff
@@ -13,33 +13,40 @@
...
     (range (rand-int 50))))
 
 
 
 
+(defn
+  stringify_map
+  [the_map]
+  (for [s (keys the_map)] (let [k s v (get the_map k)] (str k "," v " ; ")))
+  )
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: Make this a 2-liner.

(defn stringify_map [the_map]
  (for ...

You also really don't need that let, just rename s to k and do the get inline.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 4]
```diff
@@ -13,33 +13,40 @@
...
+
+
 (react/defc WorkspaceMethodsConfigurationsList
   {:render
    (fn [{:keys [props]}]
      [:div {:style {:padding "0 4em"}}
       (if (zero? (count (:method-confs props)))
         [:div {:style {:textAlign "center" :backgroundColor (:background-gray style/colors)
-                       :padding "1em 0" :borderRadius 8}}
+                       :padding   "1em 0" :borderRadius 8}}
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: Get rid of the extra space, and in all the places below

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs (hunk 2 above)
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: Yes

**Pull request merged**